하은의 real-world/haeun-real-world/src/blocks/Articles/Article.tsx

```diff
@@ -0,0 +1,52 @@
+import { ISOtoDate } from "../Calculations/DateConversion";
+import { ArticleTags } from "./ArticleTags";
+
+export type UserType = {
+    username: string,
+    bio: string,
+    image: string,
+    following: boolean
+};
+
+export type ArticleType = {
+    articles: articleType[],
+    articlesCount: number
+}
+
+export type articleType = {
+    slug: string,
+    title: string,
+    description: string,
+    body: string,
+    tagList: string[],
+    createdAt: string,
+    updatedAt: string,
+    favorited: boolean,
+    favoritesCount: number,
+    author: UserType
+};
+
+export const Article = ({article}) => {
```

Comment:
```suggestion
export const Article = ({article: {author, createdAt, favoritesCount, title, description, tagList}}) => {
```
로 하면 아래에서 article. 해서 반복을 안할수 있다!